hack/scripts/direct_lvm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# WARNING: THIS SCRIPT HAS MUTLIPLE PURPOSES.
+# TAKE CARE WHEN EDITING.
+
+# This is a scripted version of https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-manually
+
+set -ex
+
+if [[ $(id -u) != 0 ]]; then
+  echo "Run this script as root..." >&2
+  exit 1
+fi
+
+DISK_NAME="${1:-sdb}"
+DISK_PATH="/dev/$DISK_NAME"
+POOL_NAME="${1:-flintlock}"
+PROFILE="/etc/lvm/profile/$POOL_NAME-thinpool.profile"
+CROOT=/var/lib/containerd-dev
```

Comment:
Should the `/var/lib/containerd-dev` be different when using this? Perhaps `/var/lib/containerd-flintlock`?

Reply:
yep sounds good